src/daemon/rpmostreed-sysroot.cxx

```diff
@@ -808,6 +808,7 @@ rpmostreed_sysroot_prep_for_txn (RpmostreedSysroot *self, GDBusMethodInvocation
     {
       if (rpmostreed_transaction_is_compatible (self->transaction, invocation))
         {
+          sd_journal_print (LOG_INFO, "Client rejoining existing transaction");
```

Comment:
Minor: "rejoining" leads to believe it's a client we had previously seen that disconnected and then reconnected. Should we just say "joining" instead?